course/layouts/course/course_home.html

```diff
@@ -24,13 +24,13 @@
         </div>
       </div>
       {{ partial "mobile_nav_toggle.html" . }}
-      <div class="col-lg-8 course-description {{ if $shouldCollapseContent }}collapsed{{ end }}">
+      <div class="col-lg-8 course-description expand-container {{ if $shouldCollapseContent }}collapsed{{ end }}">
         <h4 class="font-weight-bold course-description-title">Course Description</h4>
         <div class="mb-1 description">{{- .Content -}}</div>
         <div class="mb-3">
           {{ if $shouldCollapseContent }}
             <button class="expand-link" aria-expanded="false">
```

Comment:
When tabbing through the elements in this page, the Read More link becomes focused but does not receive an outline.  The enter button works to toggle the expander, but it's not clear that "Read More" is selected.

Reply:
I couldn't figure out how to fix this via CSS so it's probably better to address in a separate PR. However I'm not sure it's an accessibility issue since a screenreader would still be able to focus even if the outline isn't rendering